hooks/post_user_create

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
-app="wallabag2"
+app="__APP__"
+user="__APP__"
```

Comment:
I'm randomly passing by, but I'm thinking for all these custom hooks that require to know which app we're dealing with, we could maybe use $0 (which supposedly is `50-$app`) to know the app name, which would avoid having to replace __APP__ during install / upgrades

Reply:
Should we fix this now, or merge ?
I realize this PR waited for a long time.